ingestion/src/metadata/ingestion/source/database/dbt.py

```diff
@@ -286,9 +286,6 @@ def yield_data_models(self, dbt_files: DbtFiles) -> Iterable[DataModelLink]:
             self.context.dbt_tests = {}
             for key, manifest_node in manifest_entities.items():
                 try:
-                    # Skip the analysis node since it does not contain relevant metatada
-                    if manifest_node["resource_type"] in ["analysis"]:
-                        continue
 
                     # If the run_results file is passed then only DBT tests will be processed
                     if dbt_files.dbt_run_results:
```

Comment:
```suggestion
                    if dbt_files.dbt_run_results or manifest_node["resource_type"] in ["analysis", "test"]:
```

Reply:
to make it a bit safer we could:

```python
manifest_node("resource_type") in ("analysis", "test")
```
Also, to check `in` requests tuples/sets perform better

Reply:
plus lets add some logging if we are skipping logic